### PR TITLE
DAC6-1471-Moved-the-spinner

### PR DESCRIPTION
**Pull request by shaileshp0110**

### CODE REVIEW

File: app/assets/javascripts/upload-spinner.js [hunk 1]
```diff
@@ -1,34 +1,71 @@
...
+                     console.log("status changed, updating page", data);
+                      window.location = data.url;
+                } else if (jqXhr.status === 100) {
+                 console.log("status didn't change, we not updating anything continue", jqXhr.status);
+                } else {
+                    console.log("Something went wrong", jqXhr);
+                }
+            });
         }, 3000);
+        console.log("intervalRefreshScheduled, id: ", window.refreshIntervalId);
```
Comment (ashwinirudrappa) [inline app/assets/javascripts/upload-spinner.js:68]: @shaileshp0110 Do we need these log messages

File: app/connectors/UpscanConnector.scala [hunk 2]
```diff
@@ -74,6 +71,8 @@ class UpscanConnector @Inject() (configuration: FrontendAppConfig, httpClient: H
     httpClient.GET[HttpResponse](statusUrl).map {
       response =>
         logger.debug(s"Status uploaded: $response")
+        logger.debug(s"********************\n*******Status uploaded: ${response.json}******\n****************")
```
Comment (ashwinirudrappa) [inline app/connectors/UpscanConnector.scala:74]: logs

File: app/controllers/UploadFileController.scala [hunk 3]
```diff
@@ -97,27 +91,27 @@ class UploadFileController @Inject() (
...
+                val errorReason = r.details.failureReason
+                Future.successful(Ok(Json.toJson(UpScanRedirect(routes.UploadFileController.showError("OctetStream", errorReason, "").url))))
               } else {
                 logger.debug(s"Upload rejected. Error details: ${r.details}")
-                Future.successful(Redirect(routes.NotXMLFileController.onPageLoad()))
+                Future.successful(Ok(Json.toJson(UpScanRedirect(routes.NotXMLFileController.onPageLoad().url))))
               }
             case Some(Quarantined) =>
-              Future.successful(Redirect(routes.VirusFileFoundController.onPageLoad()))
+              Future.successful(Ok(Json.toJson(UpScanRedirect(routes.VirusFileFoundController.onPageLoad().url))))
```
Comment (ashwinirudrappa) [inline app/controllers/UploadFileController.scala:105]: Every call get painted on the same page

**Review by ashwinirudrappa — APPROVED** [review]

File: app/controllers/UploadFileController.scala (hunk 3 above)
Comment (shaileshp0110) [reply app/controllers/UploadFileController.scala:105]: Yes javascript changes the location

**Pull request merged**